test/lib/cql_test_env.cc

```diff
@@ -852,9 +855,6 @@ class single_node_cql_env : public cql_test_env {
             }
 
             group0_client.init().get();
-            auto stop_system_keyspace = defer([this] {
```

Comment:
The same issue is in main, but it doesn't reveal itself because sys.ks. is not stopped there at all.
Usually "aborted start that results in stop called without shutdown" is solved by calling service::shutdown() out of service::stop(), e.g. see migration_manager::stop() vs migration_manager::drain() (drain is another name for shutdown)